toolbox/certbot/renew.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Renewing the certificates for the Airflow stack"
+/usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow.austinmobility.io
+/usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow-webhook.austinmobility.io
+/usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow-workers.austinmobility.io
+/usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow-weather.austinmobility.io
+
+# Build the HAProxy image with the renewed certificates
+cd /usr/airflow/atd-airflow
+docker compose build haproxy
```

Comment:
rather than build these into the image, would it be better/safer to mount them on start?

Reply:
i'm kind of thrown off by the the image build name seems to shadow the base `haproxy` image. i'm not saying this is actually what's happening or it's a bad pattern in any way, but it has me thinking about if we want to use github actions for building these images so that we can better keep track of which build is running on prod vs local. i realize is outside the scope of this issue and i'm happy to make an issue!

Reply:
👍 I am totally OK with changing the name of the `haproxy` service inside the `docker-compose.yml` file to differentiate it from the base image that it originates from. 

👍 If we wanted to publish the haproxy image, then I think your idea about getting the keys into it at run time and not built into the image is a must have. It's for the best to do it that way really - it makes it impossible to leak them via pushing the image.

To your question about building this image with GitHub actions, the event that the build is responding to is actually the certbot renewal event, and not a PR getting merged into a certain branch. That's not to say we can't add some `docker push` commands into the script to keep a copy of the image on dockerhub if we want to go the publishing route.

To the question of if we want to push these images up to docker hub, I think that would be OK. If the intent is to run the same images in production and in local development, we'll probably have some trouble because we'll need probably want to build multi-architecture images, and more importantly, there are a handful of changes that differ between the soon-to-be `main` branch (currently `airflow-v2`) and `production`. 

Here are some details on the differences between `production` and the main branch. `production` has [a commit](https://github.com/cityofaustin/atd-airflow/commit/a855cf95012ad84230e29a1c97bea9f9146a2ce1) ahead of the main branch, with the changes shown here: https://github.com/cityofaustin/atd-airflow/compare/airflow-v2...production. It will also accumulate merge commits ahead of the main branch as we merge the main branch into it, but that's correct as the record of merging into `production`. 

This commit ahead of the main branch contains the changes to the main branch that configure it to run in production being the port configuration and the SSL configuration. If we were to merge this back into main, we'd loose the feature of always being able to checkout the main branch and spin it up locally.

Reply:
Frank and I had a chat about some ways forward. I will open a separate issue that recaps that conversation 🙌

Reply:
I created [#12552](https://github.com/cityofaustin/atd-data-tech/issues/12552) for unifying our git/docker setup for local dev.

That issue does not mention bind-mounting the SSL certs via docker compose: We probably want to include that change in this PR?